test/command_callback/test_scss_sasslint_command_callback.vader

```diff
@@ -0,0 +1,43 @@
+Before:
+  call ale#assert#SetUpLinterTest('scss', 'sasslint')
+  call ale#test#SetFilename('test.scss')
+  unlet! b:executable
+
+After:
+  call ale#assert#TearDownLinterTest()
+
+Execute(should default to source, bin/sass-lint.js):
+  call ale#test#SetFilename('sasslint-test-files/with-source/test.scss')
+
+  let b:executable = ale#path#Simplify(
+  \   g:dir
+  \   . '/sasslint-test-files/with-source/node_modules/sass-lint/bin/sass-lint.js'
+  \)
+
+  AssertLinter b:executable,
+  \ (has('win32') ? 'node.exe ' : '')
+  \   . ale#Escape(b:executable)
+  \   . ' -v -q -f compact %t'
+
+Execute(should fallback to bin, .bin/sass-lint):
+  call ale#test#SetFilename('sasslint-test-files/with-bin/test.scss')
+
+  let b:executable = ale#path#Simplify(
+  \   g:dir
+  \   . '/sasslint-test-files/with-bin/node_modules/.bin/sass-lint'
+  \)
+
+  AssertLinter b:executable, ale#Escape(b:executable) . ' -v -q -f compact %t'
```

Comment:
The test failed on Travis CI here because you need to commit hidden directories or files with `git add -f`. Hidden files are in `.gitignore`.